test/cookbooks/osl-git-test/recipes/attributes.rb

```diff
@@ -0,0 +1,56 @@
+#
+# Cookbook:: osl-git-test
+# Recipe:: attributes
+#
+# Copyright:: 2018, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Tests git credentials when falling back to creds in attributes
+
+# Testing default properties
+git_credentials 'root'
+
+# Testing non-default properties
+user 'foo' do
+  manage_home true
+end
+
+group 'bar'
+
+# Testing non-default properties
+git_credentials 'foo' do
+  path '/home/foo/.git-credentials'
+  group 'bar'
+  mode '0400'
+end
+
+# Testing git repo clone that requires credentials
+# (Use ENV variables referenced in attributes in .kitchen.yml)
+git '/root/test' do
+  repository 'https://git.osuosl.org/osuosl/test.git'
+end
+
+git '/home/foo/test' do
+  repository 'https://git.osuosl.org/osuosl/test.git'
```

Comment:
This resource should have ``user`` set to ``foo`` to ensure it's testing that user's git store. I'm pretty sure this is using the root's store.